SPECS/libxkbcommon/libxkbcommon.spec

```diff
@@ -1,30 +1,20 @@
-%define required_wayland_version 1.2.0
-%define required_wayland_protocols_version 1.7
-
-Summary:        X.Org X11 XKB parsing library
 Name:           libxkbcommon
-Version:        1.5.0
-Release:        1%{?dist}
-License:        MIT
+Version:        1.6.0
+Release:        2%{?dist}
+Summary:        X.Org X11 XKB parsing library
+License:        MIT AND X11 AND MIT-CMU
+URL:            http://www.x.org
```

Comment:
Similar ask here to keep the original order of tags.

Reply:
same response; this is exactly the order of the inherited spec file.